test/index.js

```diff
@@ -8,8 +8,23 @@ import {Application} from 'spectron';
 let app;
 
 test.before(async () => {
+  let pathToBinary;
+
+  switch (process.platform) {
+    case 'linux':
+      pathToBinary = path.join(__dirname, '../dist/linux-unpacked/HyperTerm');
+      break;
+
+    case 'darwin':
+      pathToBinary = path.join(__dirname, '../dist/mac/HyperTerm.app/Contents/MacOS/HyperTerm');
+      break;
+
+    default:
+      throw new Error('Path to the built binary needs to be defined for this platform in test/index.js');
```

Comment:
Is it the case that you don't know where windows binaries are located to add them to this switch statement?